Added a Quick Start Guide to README (#828)
renovatebot/github-action: README.md

| <a href="https://renovatebot.com"><img alt="Renovate enabled" src="https://img.shields.io/badge/renovate-enabled-brightgreen.svg?style=flat-square"></a>                                                                | Dependencies | Renovate             |
| <a href="https://github.com/renovatebot/github-action/actions"><img alt="GitHub workflow status" src="https://img.shields.io/github/actions/workflow/status/renovatebot/github-action/build.yml?style=flat-square"></a> | Build        | GitHub Actions       |

> ## Quick Start

Reviewer: As noted, it might be worth adding the word `GitHub` here -- possibly with other words (`Workflow`, "using a"?)

```suggestion
## Quick Start with a GitHub Workflow 
```

> ### `Step 1: Renovate Explained`

Reviewer: I wouldn't call this a step.

> Renovate is a nifty tool for automatically updating your projects dependencies so you don't have to manually! Lets go through how to setup a GitHub Action for Renovate.

Reviewer: ```suggestion
Renovate is a nifty tool for automatically updating your projects dependencies so you don't have to manually! Let's go through how to set up a GitHub Action for Renovate.
```

> ### `Step 2: Generate Personal Access Token (PAT)`

Reviewer 2: DO NOT use PAT!! This is a wrong approach and it's a security and maintenability nightmare. 

You do not need any tokens in your repository since thre are company-wide token issues and managed by Developer Experience team and they are available in all repositories

We must generate a PAT for Renovate in order for it to be able to interact with your GitHub.

1. Go to your GitHub settings.

> 2. Navigate to Developer settings (the very bottom) and then select Personal access tokens (Tokens (classic)).

Reviewer 3: this should link to https://docs.renovatebot.com/modules/platform/github/ there as some more about auth for github.

but you can also use the action to run agains other platforms 😉

3. Click `Generate new token`.
4. Choose the 'repo' category at the top.
5. Click `Generate token` at the bottom (and save it for later).

### `Step 3: Add Your PAT as a Secret in Your Repo`
Now we add the PAT you created as a secret in your GitHub repo.

1. Go to your repository.
2. Navigate to Settings -> Secrets -> Actions.
3. Click `New repository secret.`
4. Name your secret `RENOVATE_TOKEN` and paste your PAT in the value field.
5. Click `Add secret`.

> ### `Step 4: Create renovate.json`

Reviewer 3: nope, not required step. renovate will onboard all accessable repos by default.

Renovate requires this .json file so lets setup a basic one.

1. Create `renovate.json` in the base of your repo.
2. Specify a base configuration:

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:base"]
}
```

> ### `Step 5: Set Up GitHub Action Workflow`

Reviewer: ```suggestion

### `Step 5: Set Up GitHub Action Workflow`
```

Here we need to setup the yaml for your Github Action Workflow.

1. Make a new directory in your repo titled `.github/workflows`.
2. Inside your new directory create a file `renovate.yml`.

> 3. Define the workflow 

Reviewer 3: Disable autodiscover and set repo explicit or set autodiscover filter for security reasons!

```yaml
name: Renovate

on:
  schedule:
    - cron: '0 3 * * *' 

jobs:
  renovate:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v3
      - name: Renovate
        uses: renovatebot/github-action@v40.0.3
        with:
          token: ${{ secrets.RENOVATE_TOKEN }}

```

> this sets up your workflow to run at 3am daily (editable) which is when Renovate will check your dependencies for updates.

Author: Thank you so much for the suggestions guys! I will follow them all and update my PR when I'm back from vacation next week.
